pkgs/development/python-modules/scp/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, pythonPackages
+, setuptools
+, paramiko
+}:
+
+buildPythonPackage rec {
+  pname = "scp";
+  version = "0.10.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "18f59e48df67fac0b069591609a0f4d50d781a101ddb8ec705f0c2e3501a8386";
+  };
+
+  propagatedBuildInputs = [
+    paramiko
+  ];
+
+  meta = with stdenv.lib; {
+    homepage = https://github.com/jbardin/scp.py;
+    description = "scp module for paramiko";
```

Comment:
Capitalize first letter (or even SCP in this case): https://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes